src/traces/indicator/attributes.js

```diff
@@ -0,0 +1,408 @@
+            ].join(' ')
+        },
+        position: {
+            valType: 'enumerated',
+            values: ['top', 'bottom', 'left', 'right'],
+            role: 'info',
+            dflt: 'bottom',
+            editType: 'plot',
+            description: [
+                'Sets the position of delta with respect to the number.'
+            ].join(' ')
+        },
+        relative: {
+            valType: 'boolean',
+            editType: 'plot',
+            role: 'info',
+            dflt: false,
+            description: [
+                'Show relative change'
+            ].join(' ')
+        },
+        valueformat: {
+            valType: 'string',
+            role: 'info',
+            editType: 'plot',
+            description: [
+                'Sets the value formatting rule using d3 formatting mini-language',
+                'which is similar to those of Python. See',
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+        increasing: {
+            symbol: {
+                valType: 'string',
+                role: 'info',
+                dflt: delta.INCREASING.SYMBOL,
+                editType: 'plot',
+                description: [
+                    'Sets the symbol to display for increasing value'
+                ].join(' ')
+            },
+            color: {
+                valType: 'color',
+                role: 'info',
+                dflt: delta.INCREASING.COLOR,
+                editType: 'plot',
+                description: [
+                    'Sets the color for increasing value.'
+                ].join(' ')
+            },
+            editType: 'plot'
+        },
+        decreasing: {
+            symbol: {
+                valType: 'string',
+                role: 'info',
+                dflt: delta.DECREASING.SYMBOL,
+                editType: 'plot',
+                description: [
+                    'Sets the symbol to display for increasing value'
+                ].join(' ')
+            },
+            color: {
+                valType: 'color',
+                role: 'info',
+                dflt: delta.DECREASING.COLOR,
+                editType: 'plot',
+                description: [
+                    'Sets the color for increasing value.'
+                ].join(' ')
+            },
+            editType: 'plot'
+        },
+        font: extendFlat({}, textFontAttrs, {
+            description: [
+                'Set the font used to display the delta'
+            ].join(' ')
+        }),
+        editType: 'calc'
+    },
+    gauge: {
+        shape: {
+            valType: 'enumerated',
+            editType: 'plot',
+            role: 'info',
+            dflt: 'angular',
+            values: ['angular', 'bullet'],
+            description: [
+                'Set the shape of the gauge'
+            ].join(' ')
+        },
+        value: extendDeep({}, gaugeBarAttrs, {
```

Comment:
Hmm. I'm not sure I like having `value` both as an attribute name and an attribute container. Maybe `bar` as in `gauge.bar.color` or `pane` (like [highcharts](https://www.highcharts.com/demo/gauge-solid)) would be better?

Reply:
I think `bar` makes sense. Maybe @plotly/plotly_js @nicolaskruchten have another idea?

Reply:
I went ahead and renamed `gauge.value` to `gauge.bar` in 95de45724b62a7a1c3d22434af838bd8c690cbfe